tedana/resources/extern_regress_dicts/Fmodel.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'd still rather merge these configs into the decision tree. Decision tree configs define necessary metrics, so we _should_ be determining what metrics to calculate from them anyway (#921). If you are concerned about column names, we could include regular expressions to capture common cases, or just let users update the "info" to say that they modified the tree to match their column names, but that the didn't change the underlying logic of the tree.

Reply:
Still thinking about this & leaning towards agreeing with you. I think we might still need an option for a regressor column config file, but if a decision tree is looking for specific column classes in order to function, then that info could be in the decision tree json. I'll play around to see how this looks in practice.

Reply:
This is now done and works with regular expressions.